src/extensions/default/JavaScriptCodeHints/node/TernNodeDomain.js

```diff
@@ -830,6 +841,7 @@ function init(domainManager) {
             }
         ]
     );
+    setTimeout(checkInterfaceAndReInit, 5000);
```

Comment:
Any specific reason for the `5000` timeout or is it just in case?

Reply:
I can reduce it to 1000 as well

Reply:
Are there any heuristics that could help one figure out when it's a good time to reinit?

Reply:
I was trying to find it out, but it's really dependent on environment. It's basically the time to re-establish NodeConnections when a node restart has happened. Tested on 3 different environments and it seems really fast. I have changed the timer duration to 1s now.

Reply:
🆗  any idea what happens if the the `NodeConnections` aren't up by that time? Hints just don't work?